src/pages/incident/IncidentDetails.tsx

```diff
@@ -119,88 +120,91 @@ export function IncidentDetailsPage() {
   }
 
   return (
-    <SearchLayout
-      contentClass="pl-6"
-      onRefresh={() => refetch()}
-      title={
-        <div className="flex my-auto">
-          <span className="text-xl flex">
-            {" "}
-            <Link to="/incidents">Incidents&nbsp;</Link>
-            {" / "}
-            {!isLoading && (
-              <div className="font-semibold">
-                <div>&nbsp;{incident.title}</div>
-              </div>
-            )}
-          </span>
-        </div>
-      }
-    >
-      <div className="flex flex-row min-h-full h-auto mt-2">
-        <div className="flex flex-col flex-1 p-6 min-h-full h-auto">
-          <div className="max-w-3xl lg:max-w-6xl w-full mx-auto">
-            {Boolean(topologyIds?.length) && (
-              <section>
-                <div className="border-b">
-                  <div className="px-2 py-2 flex flex-nowrap overflow-x-auto">
-                    {topologyIds?.map((id) => (
-                      <TopologyCard
-                        key={id}
-                        size={Size.large}
-                        topologyId={id}
-                      />
-                    ))}
-                  </div>
+    <>
+      <Head prefix="Incident Details" />
```

Comment:
Would be nice if we could have the incident title here, instead of the generic incident details title

Reply:
Ok. Will fix this